paradex_py/api/http_client.py

```diff
@@ -22,9 +22,9 @@ def request(
         self,
         url: str,
         http_method: HttpMethod,
-        params: Optional[dict] = None,
-        payload: Optional[Union[Dict[str, Any], List[Dict[str, Any]]]] = None,
-        headers: Optional[Any] = None,
+        params: dict | None = None,
+        payload: dict[str, Any] | list[dict[str, Any]] | None = None,
+        headers: Any | None = None,
```

Comment:
no one (external users) using the sdk with python 3.9 ?

Reply:
a lot of underlying libs are not supporting 3.9 - would need to migrate